doc/rados/operations/crush-map-edits.rst

```diff
@@ -1,25 +1,25 @@
-Manually editing a CRUSH Map
-============================
+Manually editing the CRUSH Map
+==============================
 
-.. note:: Manually editing the CRUSH map is an advanced
-	  administrator operation.  All CRUSH changes that are
-	  necessary for the overwhelming majority of installations are
-	  possible via the standard ceph CLI and do not require manual
-	  CRUSH map edits.  If you have identified a use case where
-	  manual edits *are* necessary with recent Ceph releases, consider
-	  contacting the Ceph developers so that future versions of Ceph
-	  can obviate your corner case.
+.. note:: Manually editing the CRUSH map is an advanced administrator
+   operation. For the majority of installations, CRUSH changes can be
+   implemented via the Ceph CLI and do not require manual CRUSH map edits.  If
```

Comment:
@zdover23 - A puzzling introduction of double spaces (it's not the product of a sentence beginning on a new line and it's nowhere to be found in the Etherpad edit).

Reply:
Fixed.